### PR TITLE
Update Prosopo repostiory link and status

### PR DESCRIPTION
**Pull request by forgetso**
Hi, we have moved to a monorepo structure and I've updated the link in your docs.

Thanks,

Chris

### CODE REVIEW

File: docs/build/build-open-source.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ In the below sections, you can find a list of different layers of the Polkadot S
 | DAO             | [SyncraDAO](https://github.com/SyncraDAO/modular-dao) ⚪, [subDAO](https://github.com/w3f-grants-archive/subDAO-contracts) ⚪, [RainbowDAO](https://github.com/RainbowcityFoundation/RainbowDAO-Protocol-Ink-milestone_1) ⚪, [MangoBox](https://github.com/Mangoboxlabs/Mangoboxink) ⚪, [MangoSale](https://github.com/Mangoboxlabs/MangoSaleV2) 🟢|                                    |
 | Identity/DID | [Dotflow](https://github.com/TheDotflow) 🟢| |
 | Oracles | [DIA WASM Oracle](https://github.com/diadata-org/dia-wasm-oracle) ⚪| |
-| Spam Protection | [Prosopo](https://github.com/prosopo-io/integration) ⚪|                                    |
+| Spam Protection | [Prosopo](https://github.com/prosopo-io/captcha) 🟢|                                    |
```
Comment (CrackTheCode016) [inline docs/build/build-open-source.md]:
```suggestion
| Spam Protection | [Prosopo](https://github.com/prosopo/captcha) 🟢|                                    |
```

**Review by CrackTheCode016 — COMMENTED** [review]
hi @forgetso, the link doesn't work for me, did you mean this one (looks like your org name changed as well):

`prosopo-io` -> `prosopo `?

Is this correct? Thank you for your PR!

**Comment by forgetso** [comment]
> hi @forgetso, the link doesn't work for me, did you mean this one (looks like your org name changed as well):
> 
> `prosopo-io` -> `prosopo `?
> 
> Is this correct? Thank you for your PR!

Hi @CrackTheCode016, that's right, good spot! I've committed your update.

**Review by CrackTheCode016 — APPROVED** [review]
gracias, thank you!

**Pull request merged**